sam-app/hello_world/app.py

```diff
@@ -1,9 +1,152 @@
+        for dimension in event["Dimensions"]:
+            if "Name" not in dimension or dimension["Name"] == "":
+                return "Invalid Dimension Name"
+            dimension["Name"] = str(dimension["Name"])
+            if "Value" not in dimension or dimension["Value"] == "":
+                return "Invalid Dimension Value"
+            dimension["Value"] = str(dimension["Value"])
+    if "Timestamp" in event:
+        try:
+            event["Timestamp"] = int(event["Timestamp"])
+        except ValueError:
+            return "Timestamp must be an integer (epoch format)."
+    if "Value" in event:
+        try:
+            event["Value"] = float(event["Value"])
+        except ValueError:
+            return "Value type must be float."
+    if "StatisticValues" in event:
+        if "SampleCount" not in event["StatisticValues"]:
+            return "Statistic Values require a Sample Count."
+        try:
+            event["StatisticValues"]["SampleCount"] = float(event["StatisticValues"]["SampleCount"])
+        except ValueError:
+            return "Invalid SampleCount"
+        if "Sum" not in event["StatisticValues"]:
+            return "Statistic Values require a Sum."
+        try:
+            event["StatisticValues"]["Sum"] = float(event["StatisticValues"]["Sum"])
+        except ValueError:
+            return "Invalid Sum"
+        if "Minimum" not in event["StatisticValues"]:
+            return "Statistic Values require a Minimum."
+        try:
+            event["StatisticValues"]["Minimum"] = float(event["StatisticValues"]["Minimum"])
+        except ValueError:
+            return "Invalid Minimum"
+        if "Maximum" not in event["StatisticValues"]:
+            return "Statistic Values require a Maximum."
+        try:
+            event["StatisticValues"]["Maximum"] = float(event["StatisticValues"]["Maximum"])
+        except ValueError:
+            return "Invalid Maximum"
+        if event["StatisticValues"]["Minimum"] > event["StatisticValues"]["Maximum"]:
+            return "Minimum cannot be greater than Maximum."
+    if "Unit" in event:
+        event["Unit"] = str(event["Unit"])
+        possibleUnits = ['Seconds','Microseconds','Milliseconds','Bytes',\
+            'Kilobytes','Megabytes','Gigabytes','Terabytes','Bits','Kilobits',\
+            'Megabits','Gigabits','Terabits','Percent','Count','Bytes/Second',\
+            'Kilobytes/Second','Megabytes/Second','Gigabytes/Second',\
+            'Terabytes/Second','Bits/Second','Kilobits/Second','Megabits/Second',\
+            'Gigabits/Second','Terabits/Second','Count/Second','None']
+        if event["Unit"] not in possibleUnits:
+            return "Unit type not available"
+    if "StorageResolution" in event:
+        try:
+            event["StorageResolution"] = int(event["StorageResolution"])
+        except ValueError:
+            return "StorageResolution type must be integer."
+    return "Validated"
+
+def formMessage(event):
```

Comment:
python convention is lowercase with underscores, not camelcase. formMessage -> form_message